### PR TITLE
Fix Bad Expires Calculation

### PR DESCRIPTION
**Pull request by dbouwman**
https://devtopia.esri.com/dc/hub/issues/193

### CODE REVIEW

File: tests/dummy/app/routes/signin.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default Route.extend({
           this.controller.transitionToRoute('secure');
         })
         .catch((err) => {
+          debugger;
```
Comment (andrewctate) [inline tests/dummy/app/routes/signin.js:29]: Probably want to lose this and its other debugger friends.

File: app/torii-adapters/arcgis-oauth-bearer.js [hunk 2]
```diff
@@ -356,13 +354,13 @@ export default EmberObject.extend({
     // However, arcgis-rest-js's UserSession and request systems
     // expect an expiry so we will simply create one set to 8 hours
     let now = Date.now();
-    let expiresIn = 8 * 60; // 8 hous
+    let expiresIn = 8 * 60 * 60; // 8 hours, in seconds
```
Comment (andrewctate) [inline app/torii-adapters/arcgis-oauth-bearer.js:357]: Is 8 hours just an arbitrary choice?

**Review by andrewctate — APPROVED** [review]
You accidentally left some debuggers in here.

File: app/torii-adapters/arcgis-oauth-bearer.js (hunk 2 above)
Comment (dbouwman) [reply app/torii-adapters/arcgis-oauth-bearer.js:357]: yes

File: tests/dummy/app/routes/signin.js (hunk 1 above)
Comment (dbouwman) [reply tests/dummy/app/routes/signin.js:29]: all that's in the dummy app :)

**Pull request merged**